bundle/DependencyInjection/NetgenEzPlatformSiteApiExtension.php

```diff
@@ -31,6 +34,7 @@ public function load(array $configs, ContainerBuilder $container)
         $activatedBundles = array_keys($container->getParameter('kernel.bundles'));
 
         $configuration = $this->getConfiguration($configs, $container);
+        assert($configuration instanceof  Configuration);
```

Comment:
Superfluous, since the method returns the `Configuration` object.

Reply:
Method signature still allows for `null`, I added this to remove the warning in PHPStorm, as a replacement for inline type hint. This should be disabled in production, so it comes at no extra cost.

Reply:
I'm more comfortable with a (inline) typehint, following the logic from PHP docs:

`Assertions should be used as a debugging feature only. You may use them for sanity-checks that test for conditions that should always be TRUE and that indicate some programming errors if not or to check for the presence of certain features like extension functions or certain system limits and features.`

You can also probably add the `@return` type hint to the method to disable the warning in PHPStorm.

This is a style preference, so you have the last say :)

Reply:
Fair point :) Changed to return type hint in https://github.com/netgen/ezplatform-site-api/pull/94/commits/a6add147ab85ac2a83e3fd75e7c1dd94bdc08e43